### ISSUE TITLE
ScreenshotHandler.java line 233

### ISSUE THREAD

**Issue by BotForHook** [firemaples/EverTranslator#45]
#### in tw.firemaples.onscreenocr.screenshot.ScreenshotHandler$3.onImageAvailable

* Number of crashes: 1
* Impacted devices: 1

There's a lot more information about this crash on crashlytics.com:
[https://fabric.io/firemaples/android/apps/tw.firemaples.onscreenocr/issues/5c6a8fa0f8b88c29634142c2?utm_medium=service_hooks-github&utm_source=issue_impact](https://fabric.io/firemaples/android/apps/tw.firemaples.onscreenocr/issues/5c6a8fa0f8b88c29634142c2?utm_medium=service_hooks-github&utm_source=issue_impact)

**Comment by firemaples** [comment]
https://console.firebase.google.com/u/2/project/ever-translator/crashlytics/app/android:tw.firemaples.onscreenocr/issues/5c6a8fa0f8b88c29634142c2?time=last-seven-days&sessionId=5C6FBBA000A2000113EC5A6229E4F338_DNE_1_v2

```
Non-fatal Exception: java.lang.Exception: Capture screen failed
       at tw.firemaples.onscreenocr.log.FirebaseEvent.logCaptureScreenFailed(FirebaseEvent.kt:104)
       at tw.firemaples.onscreenocr.state.ScreenshotTakeState$onScreenshotHandlerCallback$1.onScreenshotFailed(ScreenshotTakeState.kt:43)
       at tw.firemaples.onscreenocr.screenshot.ScreenshotHandler$3.onImageAvailable(ScreenshotHandler.java:285)
       at android.media.ImageReader$ListenerHandler.handleMessage(ImageReader.java:812)
       at android.os.Handler.dispatchMessage(Handler.java:105)
       at android.os.Looper.loop(Looper.java:164)
       at android.app.ActivityThread.main(ActivityThread.java:6673)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:769)

Caused by java.lang.OutOfMemoryError
       at android.graphics.Bitmap.nativeCreate(Bitmap.java)
       at android.graphics.Bitmap.createBitmap(Bitmap.java:1040)
       at android.graphics.Bitmap.createBitmap(Bitmap.java:994)
       at android.graphics.Bitmap.createBitmap(Bitmap.java:944)
       at android.graphics.Bitmap.createBitmap(Bitmap.java:849)
       at android.graphics.Bitmap.createBitmap(Bitmap.java:764)
       at tw.firemaples.onscreenocr.screenshot.ScreenshotHandler$3.onImageAvailable(ScreenshotHandler.java:233)
       at android.media.ImageReader$ListenerHandler.handleMessage(ImageReader.java:812)
       at android.os.Handler.dispatchMessage(Handler.java:105)
       at android.os.Looper.loop(Looper.java:164)
       at android.app.ActivityThread.main(ActivityThread.java:6673)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:769)
```

**Comment by firemaples** [comment]
Maybe fixed on #60 and should send the origin exception without wrap it.
